circle.yml

```diff
@@ -2619,15 +2619,15 @@ windows-workflow: &windows-workflow
         <<: *full-windows-workflow-filters
         name: windows-unit-tests
         executor: windows
-        resource_class: windows.medium
+        resource_class: windows.xlarge
         requires:
           - windows-build
 
     - create-build-artifacts:
         <<: *full-windows-workflow-filters
         name: windows-create-build-artifacts
         executor: windows
-        resource_class: windows.medium
+        resource_class: windows.xlarge
```

Comment:
A little unsettling that we don't know why this is required, but glad to have windows CI back to green.

Reply:
Agreed. 😕